### PR TITLE
Add flatpak release type to regex in release.version_int

### LINKED ISSUE #2966: Bodhi metrics page is broken
Both in staging and production, visiting the Bodhi metrics page throws a 500 Error:
https://bodhi.fedoraproject.org/metrics
`'NoneType' object has no attribute 'groups'`

### PR DESCRIPTION
**Pull request by sebwoj**
Fixes #2966

Signed-off-by: Sebastian Wojciechowski <s.wojciechowski89@gmail.com>

### CODE REVIEW

File: bodhi/tests/server/test_models.py [hunk 1]
```diff
@@ -867,6 +867,38 @@ def test_all_releases(self):
...
+class TestReleaseFlatpak(ModelTest):
+    """Unit test case for the ``Release`` model for flatpak releases."""
+    klass = model.Release
+    attrs = dict(
+        name=u"F11F",
+        long_name=u"Fedora 11 Flatpaks",
+        id_prefix=u"FEDORA-FLATPAK",
+        version=u'11',
+        branch=u'f11',
+        dist_tag=u"f29-flatpak",
```
Comment (bowlofeggs) [inline bodhi/tests/server/test_models.py:820]: Should we change the 11's to 29's?

Comment (sebwoj) [reply bodhi/tests/server/test_models.py:820]: I forgot to adjust each copy of DB entry from 29 to 11. But we can finally leave 29. PR updated :)

**Review by bowlofeggs — APPROVED** [review]
Thanks!

**Comment by bowlofeggs** [comment]
The ```f28-python3-integration``` test failed due to a timeout waiting on the DB to start. I've opened https://github.com/fedora-infra/bodhi/pull/3054 to raise the timeout in the hopes that will help it to happen less often.

I'll just re-run the tests locally and merge if they pass.

**Comment by bowlofeggs** [comment]
These tests pass for me locally, so I will merge this manually.

**Pull request merged**